docs/operation/disk-management.md

```diff
@@ -0,0 +1,185 @@
+# Disk Management
+
+BanyanDB provides comprehensive disk management capabilities to prevent disk space exhaustion and ensure optimal storage utilization. This includes both automatic forced retention cleanup and disk usage monitoring.
+
+## Overview
+
+BanyanDB includes a disk monitor that automatically triggers forced retention cleanup when disk usage exceeds configured thresholds. This feature helps prevent disk space exhaustion by automatically removing old data segments and snapshots.
+
+**Note**: The disk monitor with forced retention cleanup is only available for **standalone** and **data** servers. **Liaison** servers use the `*-max-disk-usage-percent` flags to throttle writes when the write queue fills up, allowing the sync process to catch up with incoming data.
```

Comment:
You only mentioned that the liasion will throttle writes, but how about data node?

I assume all writes could be throttled until more available volume space is available no matter of clean up and manual operations.